join.spec.md

```diff
@@ -362,8 +362,6 @@ Processors MUST validate that the schema contains an enum named {join__Graph}; s
 
 As described in the core specification, all of the directives and enums defined by this schema should be removed from the supergraph's [API schema](https://specs.apollo.dev/core/v0.1/#sec-Parts-of-a-Core-Schema). For example, the {join__Graph} enum should not be visible via introspection.
 
-TODO define field set somewhere?
```

Comment:
Oops, thanks, making sure to track this now at https://github.com/apollo-specs/join/issues/5